### DISCUSSION TITLE
Speed improvement for configurable products

### DISCUSSION THREAD

**Discussion by seansan** [OpenMage/magento-lts#2095] [Ideas]
speed improvement for configurable products

how about adding a solution related to this?

https://astrio.net/blog/magento-performance-optimization-configurable-products-2/

and this link http://www.jg-dev.de/magento-configurable-product-performance-boost/#.WDwzxebhA2w

Improve all grid speeds (are largest problem to date)

improve product load speed

**Comment by infabo** [comment]
Oh, I implemented this solution (slightly adjusted with additional improvements) a while back in a project where we had several hundred of options per attribute. Gave the frontend a real rocket-like boost, from ~60sec pageload down to ~2 secs (pre-fpc-timed). I'm unsure if this should be covered by magento-lts.

**Comment by seansan** [comment]
did it solve having 1000 products with 8-12 children? (typical number for
fashion)

and is the code viewable somewhere? I see only 2013 and 2014 versions ....

**Comment by seansan** [comment]
@infabo 

Can you share your code? or code reference? 

This issue is from 2016. Is anyone using this code actively in production and can you share the results?
